app/components/UserWidget/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+import { createStructuredSelector } from 'reselect';
+import { removeToken, loggedIn, loggedOut } from 'components/AuthUser/lib';
+import { logout } from 'components/AuthUser/actions';
+import { selectGivenName, selectPicture } from 'components/AuthUser/selectors';
+import UserBadge from 'components/UserBadge';
+import styles from './styles.css';
+import buttonStyles from 'components/Button/styles.css';
+
+class UserWidget extends React.Component { // eslint-disable-line react/prefer-stateless-function
```

Comment:
OK I'll move this one. And I'm not sure where the `AuthUser` folder should go? It has reducers etc, but nothing to render.